ghost/i18n/locales/pt-BR/comments.json

```
@@ -5,44 +5,44 @@
     "{{amount}} mins ago": "{{amount}} minutos atrás",
     "{{amount}} more": "{{amount}} mais",
     "1 comment": "1 comentário",
-    "Add comment": "Adicionar comentário",
+    "Add comment": "Comentar",
     "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Adicione contexto ao seu comentário, compartilhe seu nome e expertise para promover uma discussão saudável.",
-    "Add reply": "Adicionar resposta",
+    "Add reply": "Responder",
     "Already a member?": "Já é um membro?",
     "Anonymous": "Anônimo",
-    "Are you sure?": "",
+    "Are you sure?": "Tem certeza?",
     "Become a member of {{publication}} to start commenting.": "Torne-se um membro de {{publication}} para começar a comentar.",
     "Become a paid member of {{publication}} to start commenting.": "Torne-se um membro pago de {{publication}} para começar a comentar.",
     "Cancel": "Cancelar",
     "Comment": "Comentário",
     "Complete your profile": "Complete seu perfil",
     "Delete": "Deletar",
-    "Deleted": "",
+    "Deleted": "Deletado",
     "Deleted member": "Membro deletado",
-    "Deleting": "",
+    "Deleting": "Deletando",
     "Discussion": "Discussão",
     "Edit": "Editar",
     "Edit this comment": "Editar este comentário",
     "edited": "editado",
     "Enter your name": "Digite seu nome",
-    "Expertise": "Especialidade",
+    "Expertise": "Expertise",
```

Considering that not everyone uses words with a technical meaning, I don't think “expertise” is a suitable term.
```suggestion
    "Expertise": "Especialidade",
```